pkg/collector/corechecks/servicediscovery/usm/service_test.go

```diff
@@ -619,12 +619,13 @@ func TestExtractServiceMetadata(t *testing.T) {
 				t.Skip("Not supported on Windows")
 			}
 
-			var fs fs.SubFS
-			fs = RealFs{}
+			var f fs.SubFS
```

Comment:
Why was this variable name changed?

Reply:
my IDE showed warning that name collides with imported package, I restored name as it was